lms/static/scripts/frontend_apps/components/BookPicker.tsx

```diff
@@ -69,43 +69,53 @@ export default function BookPicker({
   const canSubmit =
     (step === 'select-book' && book) || (step === 'select-chapter' && chapter);
 
-  // Provide a spacious modal UI when listing chapters (taller modal)
-  const showChapters = step === 'select-chapter' && !error;
+  /**
+   * Prevent submission of content form, which can raise an error because
+   * content has not been selected.
+   */
+  const onCancelButtonClick = (event: Event) => {
+    event.preventDefault();
+    event.stopPropagation();
+    onCancel();
+  };
 
   return (
     <Modal
       // Opt out of Modal's automatic focus handling; route focus manually in
       // sub-components
-      initialFocus={null}
-      onCancel={onCancel}
-      contentClass={classnames('LMS-Dialog LMS-Dialog--wide', {
-        'LMS-Dialog--tall': showChapters,
```

Comment:
Modal content height management is now applied to the content instead of to the Modal. Documented in the [pattern library](http://localhost:8001/ui-playground/feedback-modal).